crates/matrix-sdk-indexeddb/src/crypto_store.rs

```diff
@@ -1212,6 +1274,42 @@ impl Drop for IndexeddbCryptoStore {
     }
 }
 
+/// The objects we store in the gossip_requests indexeddb object store
+#[derive(Debug, serde::Serialize, serde::Deserialize)]
+struct GossipRequestIndexedDbObject {
+    /// encrypted hash of the SecretInfo structure
```

Comment:
```suggestion
    /// Encrypted hash of the [`SecretInfo`] structure.
```

Reply:
(sorry, my crappy formatting in doc comments is a terrible habit)